2-library/hexagonal-lib/pom.xml

```
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.6core.lib</groupId>
+    <artifactId>hexagonal-lib</artifactId>
```

bad naming of groupId + artifactId. 
Why do we need postfix `lib` if we already have it in groupdId? 
hexagonal-markers, hexagonal-annotations, hexagonal-specifications, ....
```suggestion
    <groupId>com.6core.lib</groupId>
    <artifactId>hexagonal-lib</artifactId>
```

---

Have changed to hexagonal-annotations.